docs/2.guide/2.directory-structure/1.server.md

```diff
@@ -315,8 +315,8 @@ export default defineEventHandler((event) => {
 ### Sending Redirect
 
 ```ts [server/api/foo.get.ts]
-export default defineEventHandler((event) => {
-  return sendRedirect(event, '/path/redirect/to', 302)
+export default defineEventHandler(async (event) => {
+  return await sendRedirect(event, '/path/redirect/to', 302)
```

Comment:
It's fine to return the `sendRedirect` because that means the function returns the promise, and it will be awaited.

The issue is only if it's not returned.

Reply:
We could do this so it's more obviously copy/pasteable.

```suggestion
  await sendRedirect(event, '/path/redirect/to', 302)
```